ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -245,7 +245,7 @@ static int ompi_osc_rdma_component_register (void)
                                             MCA_BASE_VAR_SCOPE_GROUP, &mca_osc_rdma_component.locking_mode);
     OBJ_RELEASE(new_enum);
 
-    ompi_osc_rdma_btl_names = "openib,ugni,uct,ucp";
+    ompi_osc_rdma_btl_names = "ugni,uct,tcp";
```

Comment:
should ofi be in this list?

Reply:
Good question.  I defer to @hjelmn to answer that...